conda_build.recipe/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+
+# Set the version info based on python version.
+VERSION="$(python setup.py --version)"
+
+# Add the git hash if it is missing.
+if [[ "${VERSION}" == *+ ]];
+then
+    VERSION="${VERSION}$(git rev-parse --short HEAD)"
```

Comment:
Yeah, it's a bit weird, but it was happening to me sometimes when I wasn't on a tag.